geometry/geometry.py

```diff
@@ -134,25 +134,27 @@ def max_parts(self, num_cuts: float) -> float:
 
         >>> circle = Circle(5)
         >>> circle.max_parts(0)
-        1.0
+        1
         >>> circle.max_parts(7)
-        29.0
+        29
         >>> circle.max_parts(54)
-        1486.0
+        1486
         >>> circle.max_parts(22.5)
-        265.375
+        Traceback (most recent call last):
+            ...
```

Comment:
What's the need for that ?

Reply:
Same reason - whole number needed, not float.